%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% KOMA-Script Presentation
% LaTeX Template
% Version 1.1 (18/10/15)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original Authors:
% Marius Hofert (marius.hofert@math.ethz.ch)
% Markus Kohm (komascript@gmx.info)
% Described in the PracTeX Journal, 2010, No. 2
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------
% KOMA-Script_MDPI_jpm-1309345
% compiling only on TexLive 2019

\documentclass[
paper=landscape,
paper=160mm:90mm, %128mm:96mm, % The same paper size as used in the beamer class
fontsize=11pt, % Font size
pagesize, % Write page size to dvi or pdf
parskip=half-, % Paragraphs separated by half a line
]{scrartcl} % KOMA script (article)

\linespread{1.12} % Increase line spacing for readability



%~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
% from main.tex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Tex
\maxdeadcycles=1000 % Output loop---200 consecutive dead cycles.

\usepackage{floatrow} % for side caption (beside)
%\floatsetup[widefigure]{margins=hangleft,capposition=beside,capbesideposition={center,outside},floatwidth=\textwidth} % {center,outside}
\usepackage{amsmath,amsfonts}
\usepackage{tikz-imagelabels} % for  tikz
%\usepackage[]{caption} % bf
%\newcommand{\bcaption}[2]{\caption{\textbf{#1} #2}}
%\usepackage{outlines}
% mark in blue or red
%\usepackage{xcolor}

%\usepackage{xr-hyper}
\usepackage{hyperref}
%\newcommand{\R}[1]{\label{#1}\linelabel{#1}} % for \label page and line
%\newcommand{\R}[1]{\linelabel{#1}} % for \label page and line
%\newcommand{\lr}[1]{page~\pageref{#1} (line~\lineref{#1})} % for \ref page and line
% for xr in your preamble
% https://www.overleaf.com/learn/how-to/Cross_referencing_with_the_xr_package_in_Overleaf
%\makeatletter
%\newcommand*{\addFileDependency}[1]{% argument=file name and extension
%  \typeout{(#1)}
%  \@addtofilelist{#1}
%  \IfFileExists{#1}{}{\typeout{No file #1.}}
%}
%\makeatother

%\newcommand*{\myexternaldocument}[1]{%
%    \externaldocument{#1}%
%    \addFileDependency{#1.tex}%
%    \addFileDependency{#1.aux}%
%}
% my external document I would like to reference the labels of. 
%\myexternaldocument{Supplementary_figures} %.tex .aux

\usepackage{array}
\usepackage{ragged2e}
\usepackage{rotating}
\usepackage{tabularx} % for resizebox?
\usepackage{makecell}
% \usepackage{array}
\usepackage{multirow}
\usepackage{colortbl}
\usepackage{hhline}
\usepackage{siunitx} % for  1e-10 scientific notation
%\usepackage{caption}
%\usepackage{subcaption}
%\usepackage{subcaption} % for figure  side by side, subfigure

%DIF 140a141
\usepackage{pbox} %DIF > 
%DIF -------

\usepackage{booktabs, multirow} % for borders and merged ranges
\usepackage{soul}% for underlines
%\usepackage[table]{xcolor} % for cell colors
\usepackage{changepage,threeparttable} 

%%% for abbreviations, or acronyms
\usepackage[acronym, nopostdot]{glossaries}  % automake
\usepackage{glossary-inline}
%\setacronymstyle{long-short}
%\renewcommand*{\glossarysection}[2][]{} 
%\renewcommand*{\glossarysection}[2][]{\textbf{#1}: }
% for abbreviations environment
%\newcommand{\abbrlabel}[1]{\makebox[3cm][l]{\textbf{#1}\ \dotfill}}
\newenvironment{abbreviation}


%\section{abbreviation}
%%%%%%%%%%%%% Define abbreviation
%\makeglossaries %https://tex.stackexchange.com/questions/110095/list-of-acronyms-is-not-displayed



\newacronym{ncbi}{NCBI}{National Center for Biotechnology Information}
\newacronym{degs}{DEGs}{differentially expressed genes}

\newacronym{ihc}{IHC}{immunohistochemistry}
\newacronym{fdr}{FDR}{false discovery rate}

\newacronym{hpa}{HPA}{the Human Protein Atlas}
\newacronym{hnscc}{HNSCC}{head and neck squamous cell carcinoma}
\newacronym{tcga}{TCGA}{the Cancer Genome Atlas}
\newacronym{tcpa}{TCPA}{the Cancer Proteome Atlas}
\newacronym{rna}{RNA}{ribonucleic acid}
\newacronym{rnaseq}{RNA-Seq}{RNA sequencing}
\newacronym{lncrna}{lncRNA}{long non-coding RNA}
%\newacronym{km}{KM}{Kaplan--Meier}
\newacronym{rppa}{RPPAs}{reverse-phase protein arrays}
\newacronym{rpma}{RPMA}{reverse-phase protein lysate microarray}

\newacronym{mmp}{MMP}{matrix metalloproteinase}
 %DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, and DKC1
 %ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, KIAA1683, and NPB

\newacronym{DKK1}{DKK1}{dickkopf WNT signaling pathway inhibitor 1} 
\newacronym{CAMK2N1}{CAMK2N1}{calcium/calmodulin dependent protein kinase II inhibitor 1} 
\newacronym{CALML5}{CALML5}{calmodulin like 5}

\newacronym{STC2}{STC2}{stanniocalcin 2} 
\newacronym{PGK1}{PGK1}{phosphoglycerate kinase 1} 
\newacronym{SURF4}{SURF4}{surfeit 4} 
\newacronym{USP10}{USP10}{ubiquitin specific peptidase 10} 
\newacronym{NEDD4}{NEDD4}{neural precursor cell expressed, developmentally down-regulated 4}
\newacronym{NDFIP1}{NDFIP1}{NEDD4 family interacting protein 1} 
\newacronym{FOXA2}{FOXA2}{forkhead box A2} 
\newacronym{STIP1}{STIP1}{stress-induced-phosphoprotein 1} 
\newacronym{DKC1}{DKC1}{dyskeratosis congenita 1, dyskerin} 

\newacronym{ZNF557}{ZNF557}{zinc finger protein 557} 
\newacronym{ZNF266}{ZNF266}{zinc finger protein 266} 
\newacronym{IL19}{IL19}{interleukin 19} 
\newacronym{MYO1H}{MYO1H}{myosin 1H} 
\newacronym{FCGBP}{FCGBP}{Fc fragment of IgG binding protein} 
\newacronym{LOC148709}{LOC148709}{LncRNA LOC148709} 
\newacronym{EVPLL}{EVPLL}{envoplakin-like protein} 
\newacronym{PNMA5}{PNMA5}{paraneoplastic antigen like 5} 
%\newacronym{KIAA1683}{KIAA1683}{IQCN, IQ Motif Containing N} 
\newacronym{IQCN}{IQCN}{IQ motif containing N} % previous name KIAA1683
% "IQ'' refers to the first two amino acids of the motif: isoleucine (commonly) and glutamine (invariably)
\newacronym{NPB}{NPB}{neuropeptide B} 

 \newacronym{rt}{RT}{radiation therapy}
 \newacronym{nccn}{NCCN}{National Comprehensive Cancer Network}
 \newacronym{hif}{HIF}{hypoxia-inducible factor}
 \newacronym{egfr}{EGFR}{epidermal growth factor receptor}
 \newacronym{ras}{RAS}{rat sarcoma}
 \newacronym{hras}{HRAS}{Harvey rat sarcoma viral oncoprotein}
 \newacronym{erk}{ERK}{extracellular signal-regulated kinases}
 \newacronym{us}{US}{United States}
 \newacronym{fda}{FDA}{Food and Drug Administration}
 \newacronym{tpf}{Tax-PF}{docetaxel, cisplatin, and 5-fluorouracil}
 \newacronym{tki}{TKI}{tyrosine kinase inhibitor}
 \newacronym{her}{HER}{human epidermal growth factor receptor}
 \newacronym{ici}{ICI}{immune-checkpoint inhibitor}
 \newacronym{ctla4}{CTLA-4}{cytotoxic T lymphocyte antigen 4}
 \newacronym{pd1}{PD-1}{programmed death 1}
 \newacronym{pdl1}{PD-L1}{programmed death ligand 1}
 \newacronym{tim3}{TIM-3}{T-cell immunoglobulin mucin protein 3}
 \newacronym{lag3}{LAG-3}{lymphocyte activation gene 3}
 \newacronym{ifng}{IFN-$\gamma$}{interferon gamma}
 \newacronym{tigit}{TIGIT}{T cell immunoglobin and immunoreceptor tyrosine-based inhibitory motif}
 \newacronym{gitr}{GITR}{glucocorticoid-induced tumor necrosis factor receptor}
 \newacronym{vista}{VISTA}{V-domain Ig suppressor of T-cell activation}
 \newacronym{tmsb4x}{TMSB4X}{thymosin beta-4 X-linked}
 \newacronym{emt}{EMT}{epithelial-mesenchymal-transition}
 \newacronym{gdc}{GDC}{Genomic Data Commons}
 \newacronym{nci}{NCI}{the National Cancer Institute}
 \newacronym{gdac}{GDAC}{genome data analysis center}
 \newacronym{rest}{REST}{Representational State Transfer} 
 \newacronym{api}{API}{application programmable interface}
\newacronym{grch38}{GRCh38}{Genome Reference Consortium Homo sapiens genome assembly 38}
\newacronym{fpkm}{FPKM}{Fragments per kilobase per million reads mapped}
\newacronym{rsem}{RSEM}{RNA-Seq by Expectation-Maximization}
\newacronym{slca}{SLC35E2A}{solute carrier family 35 member E2A}
\newacronym{slcb}{SLC35E2B}{solute carrier family 35 member E2B}
\newacronym{cde}{CDE}{Common Data Element}
\newacronym{id}{ID}{identification}
\newacronym{ajcc}{AJCC}{the American Joint Committee on Cancer}
\newacronym{uicc}{UICC}{he Union for International Cancer Control}
\newacronym{tnm}{TNM}{the tumor size (T), cervical lymph node metastases (N), and distal metastasis status (M)}
\newacronym{ci95}{95\% CI}{95\% confidence interval}
\newacronym{os}{OS}{overall survival}
\newacronym{hr}{HR}{hazard ratio}
\newacronym{hpv}{HPV}{human papillomavirus}
\newacronym{ene}{ENE}{extra-nodal extension}
\newacronym{lvsi}{LVSI}{lymph-vascular space invasion}
\newacronym{pni}{PNI}{perineural invasion}
\newacronym{doi}{DOI}{depth of invasion}
\newacronym{lnd}{LND}{lymph node density}
\newacronym{wpoi5}{WPOI-5}{worst pattern of invasion score 5}
\newacronym{glut4}{GLUT4}{glucose transporter 4}
\newacronym{slc2a4}{SLC2A4}{solute carrier family 2 member A4}
\newacronym{trim24}{TRIM24}{tripartite motif-containing 24}
\newacronym{til}{TIL}{tumor-infiltrating lymphocytes}
\newacronym{tmb}{TMB}{tumor mutational burden}




%------------------------------------------------
\usepackage{outlines}
\usepackage[font={small}]{caption}
\newcommand{\bcaption}[2]{\caption{\textbf{#1} #2}}


\usepackage[margincaption]{sidecap}

\usepackage{subcaption} % for figure  side by side, subfigure

%%%%%% updated 2020 by esdd
%% https://tex.stackexchange.com/questions/547723/latex-throws-errors-on-section
%------------------------------------------------
% Colors
\usepackage[]{xcolor}  % table % Required for custom colors
% Define a few colors for making text stand out within the presentation
\definecolor{mygreen}{RGB}{44,85,17}
\definecolor{myblue}{RGB}{34,31,217}
\definecolor{mybrown}{RGB}{194,164,113}
\definecolor{myred}{RGB}{255,66,56}
% Use these colors within the presentation by enclosing text in the commands below

\newcommand*{\mygreen}[1]{\textcolor{mygreen}{#1}}
\newcommand*{\myblue}[1]{\textcolor{myblue}{#1}}
\newcommand*{\mybrown}[1]{\textcolor{mybrown}{#1}}
\newcommand*{\myred}[1]{\textcolor{myred}{#1}}

\definecolor{asparagus}{rgb}{0.53, 0.66, 0.42}

\newenvironment{MyColorPar}[1]{% for RED marked of revision
    \leavevmode\color{#1}\ignorespaces%
}{%
}%


% Margins
\usepackage[ % Page margins settings
  includeheadfoot,
  top=-1mm,
  bottom=1.5mm,
  left=1mm,
  right=3.5mm,
  headsep=0mm,
  footskip=8.5mm
]{geometry}

% Fonts
\usepackage[T1]{fontenc}     % For correct hyphenation and T1 encoding
\usepackage{lmodern} % Default font: latin modern font
%\usepackage{fourier} % Alternative font: utopia
%\usepackage{charter} % Alternative font: low-resolution roman font
\renewcommand{\familydefault}{\sfdefault} % Sans serif - this may need to be commented to see the alternative fonts

% Various required packages
\usepackage{amsthm} % Required for theorem environments
\usepackage{bm} % Required for bold math symbols (used in the footer of the slides)
\usepackage{tikz} % Required for colored boxes, loads graphicx and other packages
\usepackage{booktabs} % Required for horizontal rules in tables
\usepackage{multicol} % Required for creating multiple columns in slides
\usepackage{lastpage} % For printing the total number of pages at the bottom of each slide

\usepackage{microtype} % Better typography

% Slide layout configuration
\usepackage[automark]{scrlayer-scrpage} % Required for customization of the header and footer
\clearpairofpagestyles % Remove the default header and footer
\AddLayersAtBeginOfPageStyle{scrheadings}{headerbg,footerbg}% add background layers
\setkomafont{pageheadfoot}{\normalfont\sffamily} % Font settings for the header and footer
\setkomafont{pagehead}{\color{white}}
% Header configuration - if you don't want a header remove this block
\DeclareNewLayer[
  background,
  head,
  hoffset=0pt,
  width=\paperwidth,
  mode=picture,
  contents=\putLL{\myblue{\rule{\layerwidth}{\layerheight}}}
]{headerbg}

\ihead{\rightbotmark}

% Footer configuration
\KOMAoptions{footwidth=\textwidth+2mm:0pt}
\setkomafont{pagefoot}{\color{black}\tiny} % Small font size for the footnote
%\DeclareNewLayer[
  %background,
  %foot,
  %hoffset=0pt,
  %width=\paperwidth,
  %mode=picture,
  %contents=\putLL{\myblue{\rule{\layerwidth}{\layerheight}}}
%]{footerbg}
\ifoot{\myauthor\ \raisebox{0.2mm}{$\bm{\vert}$}\ \myuni} % Left side text
\ofoot*{\pagemark/\pageref{LastPage}} % Right side

% Sets vertical centering of slide contents with increased space between paragraphs/lists
\makeatletter
\renewcommand*{\@textbottom}{\vskip \z@ \@plus 1fil}
\newcommand*{\@texttop}{\vskip \z@ \@plus .5fil}
\setparsizes{1em}{\z@\@plus .25fil}{0pt plus 1fil}
\makeatother

% Remove page numbers and the dots leading to them from the outline slide
\DeclareTOCStyleEntries[linefill=\hfill,pagenumberbox=\gobble]{section}{section,subsection,subsubsection}
%\newcommand*\gobble[1]{}
\AfterTOCHead{\small}

%\renewcaptionname{english}{\contentsname}{Outline} % contentsname % Change the name of the table of contents
% Section spacing - deeper section titles are given less space due to lesser importance
\RedeclareSectionCommands[beforeskip=0pt,afterskip=0pt,afterindent=true,runin=false]{section,subsection,subsubsection}
\RedeclareSectionCommand[afterskip=-1mm]{subsection}
\RedeclareSectionCommand[afterskip=-2mm]{subsubsection}
\setcounter{secnumdepth}{\partnumdepth} % How deep sections are numbered

% Theorem style
\newtheoremstyle{mythmstyle} % Defines a new theorem style used in this template
  {0.5em} % Space above
  {0.5em} % Space below
  {} % Body font
  {} % Indent amount
  {\sffamily\bfseries} % Head font
  {} % Punctuation after head
  {\newline} % Space after head
  {\thmname{#1}\ \thmnote{(#3)}} % Head spec

\theoremstyle{mythmstyle} % Change the default style of the theorem to the one defined above
\newtheorem{theorem}{Theorem}[section] % Label for theorems
\newtheorem{remark}[theorem]{Remark} % Label for remarks
\newtheorem{algorithm}[theorem]{Algorithm} % Label for algorithms


% The code for the box which can be used to highlight an element of a slide (such as a theorem)
\newcommand*{\mybox}[2]{% The box takes two arguments: width and content
  \par\noindent
  \begin{tikzpicture}[mynodestyle/.style={rectangle,draw=myblue,thick,inner sep=2mm,text justified,top color=white,bottom color=white,above}]
    \node[mynodestyle,at={(0.5*#1+2mm+0.4pt,0)}]{% Box formatting
      \begin{minipage}[t]{#1}
      #2
      \end{minipage}%
    };
  \end{tikzpicture}
\par\vspace{-1.3em}}

%----------------------------------------------------------------------------------------
%	PRESENTER INFORMATION
%----------------------------------------------------------------------------------------

\newcommand*{\mytitle}{Transcriptomic Analysis for Prognostic Value in Head and Neck Squamous Cell Carcinoma} % Title
\newcommand*{\runninghead}{HNSCC Biomarker} % Running head displayed on almost all slides
\newcommand*{\myauthor}{Tex Li-Hsing Chi} % Presenters name(s)
\newcommand*{\mydate}{\today} % Presentation date
\newcommand*{\myuni}{Taipei Medical University} % University or department

%----------------------------------------------------------------------------------------

\begin{document}

%----------------------------------------------------------------------------------------
%	TITLE SLIDE
%----------------------------------------------------------------------------------------

% Title slide - you may have to tweak a few of the numbers if you wish to make changes to the layout
\thispagestyle{empty} % No slide header and footer
\begin{tikzpicture}[remember picture,overlay] % Background box
  \node [xshift=\paperwidth/2,yshift=\paperheight/2] at (current page.south west)
    [rectangle,fill,inner sep=0pt,minimum width=\paperwidth,minimum height=\paperheight/2.1,top color=myblue,bottom color=myblue]{}; % Change the height of the box, its colors and position on the page here
\end{tikzpicture}
% Text within the box
\begin{flushright}
  \vspace{1.6cm}
  \color{white}\sffamily
  {\bfseries\Large\mytitle\par}% Title
  \vspace{0.5cm}
  \normalsize
  \myauthor\par % Author name
  \mydate\par % Date
  \myuni\par
  \vfill
\end{flushright}

\clearpage
%----------------------------------------------------------------------------------------
%	TABLE OF CONTENTS
%----------------------------------------------------------------------------------------

\thispagestyle{empty} % No slide header and footer

\small\tableofcontents % Change the font size and print the table of contents - it may be useful to shrink the font size further if the presentation is full of sections
% To exclude sections/subsections from the table of contents, put an asterisk after \(sub)section like so: \section*{Section Name}

\clearpage

%----------------------------------------------------------------------------------------
%	PRESENTATION SLIDES
%----------------------------------------------------------------------------------------

%\section*{Vize}

%\clearpage

%------------------------------------------------



%------------------------------------------------


\section*{Verbatim}

How to include a theorem in this presentation:
\begin{verbatim}
\mybox{0.8\textwidth}{
\begin{theorem}[Murphy (1949)]
Anything that can go wrong, will go wrong.
\end{theorem}
}
\end{verbatim}

\clearpage

%------------------------------------------------



\subsection*{Figure}

%\clearfloatsetup{figure}
%\floatsetup[figure]{style=no,capposition=beside,capbesideposition={center,inside},capbesideframe=yes,facing=yes}

\begin{figure}[ht]
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,center},capbesidewidth=.35\linewidth,capbesidesep=quad}}]{figure}[\FBwidth]
{\caption{\texttt{capbesidesep=quad}. I want to thank Sonia. The caption can contain any text but needs to describe the image with enough detail for a reader to completely understand the image.}}
{\includegraphics[width=0.6\textwidth]{placeholder.jpg}}
\end{figure}
%\includegraphics[width=9cm]{placeholder}

%\sidecaptionvpos{c}
%\caption{I want to thank Sonia.}
%\end{captionbeside}


\clearpage

%------------------------------------------------



\section{Introduction}


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage


% figure replaced by minipage
%\clearfloatsetup{figure}
%\floatsetup[figure]{style=no,capposition=beside,capbesideposition={center,outside},capbesideframe=yes,facing=yes}

\begin{figure}[ht]
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,center},capbesidewidth=.35\linewidth,capbesidesep=quad}}]{figure}[\FBwidth]
%\centering
%\widefigure, keepaspectratio
{\includegraphics[width=7cm]{Figure_1_manuscript_workflow.pdf}} % .PDF is better than .png
%, height=8cm
%\caption % , step 1 (\textcolor{blue}{blue line}: main procedure) and step 2 (\textcolor{orange}{orange line}: analysis export).
%Step 3 (purple line: dealing with surgical margin).
%\sidecaptionvpos{c}
{\caption{\hl{A workflow of \acrshort{hnscc} biomarker discovery.}
The workflow includes data retrieval from the TCGA GDC data portal, data processing with merging and cleaning, and then performing the survival analyses (within \textcolor{yellow}{yellow} square).}} %The Cutoff engine (in R script: cutofFinder\_func.HNSCC.R, a serial cutoff for grouping patients with \textcolor{asparagus}{low} or \textcolor{red}{high} expression of a specific gene, to yield a collection of \protect\textit{P} values; please see Materials and Methods section for details) might calculate all possible Kaplan--Meier \protect\textit{P} values (corrected by \acrlong{fdr}, FDR, method) to find the optimal cutoff value of gene expression for subsequent Cox modeling. The candidate selection performs (1) dissecting and selection of candidate genes with further Bonferroni adjusted \protect\textit{P} values and the hazard ratios of a Cox model, based on the results from the survival analyses; (2) survival analyses of the other HNSCC dataset (GSE65858) using Kaplan--Meier estimates (with FDR corrections) and Cox modeling.\\ The biomarker candidates were consensus results of TCGA and GSE65858. (HNSCC: head and neck squamous cell carcinoma; TCGA: the Cancer Genome Atlas; RNA-Seq: RNA sequencing; GDC: Genomic Data Commons.)} } % end of caption

% Description:1) FDR correction of Kaplan--Meier \protect\textit{P} values during Cutoff finding; and 2) Bonferroni correction of Kaplan--Meier \protect\textit{P} values after Cox modeling for candidate selection.

%\end{minipage}
\end{figure}

\clearpage

\section{Results}


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage
\subsection{TCGA HNSCC}

\begin{figure}[hbt!]
\centering
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,center},capbesidewidth=.35\linewidth,capbesidesep=quad}}]{figure}[\FBwidth]
{\includegraphics[width =\textwidth, height = 0.6\textheight, keepaspectratio]{Figure2.pdf}}
{\bcaption{The initial progress of candidate selection from the TCGA \acrshort{hnscc} cohort.}{The \protect\textit{P} value of Kaplan--Meier survival was one of the selection criteria.
The effect size was estimated by Cox's hazard ratio.
Initial trial step: (a) univariate HR versus FDR-adjusted \protect\textit{P} value; (b) multivariate HR versus FDR-adjusted \protect\textit{P} value.
After stringent restriction by Bonferroni-adjusted \protect\textit{P} values and Cox's HR, a few top-ranked genes are shown by
(c) univariate HR versus Bonferroni-adjusted \protect\textit{P} value;  (d) multivariate HR versus Bonferroni-adjusted \protect\textit{P} value.
(TCGA: \acrlong{tcga}; HR: hazard ratio; FDR: \acrlong{fdr})}}
\end{figure}

\clearpage

\begin{figure}[hbt!]
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,center},capbesidewidth=.35\linewidth,capbesidesep=quad}}]{figure}[\FBwidth]
% TCGA FDR Pvalue of CAMK2N1 (1.628308e-05), IL19 ( 6.543871e-06), FCGBP (4.827833e-05), CALML5 (0.0001970348)
{\setlength{\unitlength}{1cm}
\begin{picture}(15, 20) %(1,0.55038404)%
\centering
  \put(0,0){\includegraphics[width=6cm]{Figure_4_CAMK2N1_CALML5_FCGBP.pdf}}%
  \put(2.5, 14.3){\fontfamily{qcr}\selectfont
  \tiny *\protect\textit{P} = \num{1.63e-05}}%CAMK2N1
    \put(2.5, 9.45){\fontfamily{qcr}\selectfont
  \tiny *\protect\textit{P} = \num{1.97e-4}}%IL19 6.54e-06 -> CALML5 0.0001970348
    \put(2.5, 4.55){\fontfamily{qcr}\selectfont
  \tiny *\protect\textit{P} = \num{4.83e-05}}%FCGBP

%\begin{annotationimage}{width=15cm}{Figure_4_CAMK2N1_IL19_FCGBP.pdf}
%\includegraphics[width=15cm]{Figure_4_CAMK2N1_IL19.pdf}
%\draw[annotation left = {Aries at 0.3}]

%\end{annotationimage}
\end{picture}%
}

{\bcaption{Kaplan--Meier survival analyses, by cutoff finding.}
{The Kaplan--Meier curves of (a) CAMK2N1, (c) CALML5, and (e) FCGBP with optimal \protect\textit{P} values. 
((b) the cutoffs derived from it's cumulative \protect\textit{P} value plot).
%(c) Kaplan--Meier plot of IL19 under optimal \protect\textit{P} value;
((d) the cutoffs derived from it's cumulative \protect\textit{P} value plot).
(e) Kaplan--Meier plot of FCGBP under optimal \protect\textit{P} value.\\
%The optimal cutoff value for CAMK2N1 is 
The cutoffs in the cumulative \protect\textit{P} value plots of (b) CAMK2N1, (d) CALML5, and (f) FCGBP, respectively,
show that over 50\% of those unadjusted \protect\textit{P} values derived by the sliding-window cutoff-finding procedure are below 0.001.\\
(* \protect\textit{P}: \protect\textit{P} value adjusted by \acrlong{fdr}, \acrshort{fdr})
}}
\end{figure}

\clearpage

%%

\begin{figure}[H]
    %\centering
    \includegraphics[width=13cm]{Rplot_TCGA_HNSCC_CoxHR_CAMK2N1_top3FDRKM.pdf}
    \bcaption{\hl{A volcano plot of genes in survival analyses of TCGA HNSCC}.}{
%X axis: unadjusted \textit{p}~value of Kaplan--Meier survival (-log10 transformed).
%Y axis: multivariate hazard ratio from Cox proportional regression model.
%Dotted line: significant Bonferroni corrected \textit{p}~value. 
%\textcolor{red}{Red dots} mark 10 genes (unvalidated), which impact on poor prognosis ($HR>=1.5$). \textcolor{green}{Green dots} mark 10 genes (unvalidated), which affect on better survival ($HR<=0.5$).
    This cohort was applied for exploration of the candidate biomarkers.
    A total of 9416 genes had %\acrshort{fdr}-
    unadjusted \textit{p}~values of less than 0.05.
    \textcolor{red}{CAMK2N1}, \textcolor{red}{CALML5}, \textcolor{red}{FCGBP}, and 17 other genes (marked in \textcolor{black}{black square}) had hazard ratios (HRs) >$1.8$ or <$0.6$.
    The 22 genes, listed on the side, had hazard ratios between 0.6 and 1.5.
    \textcolor{red}{Red spots}: $HR$ > 1.0.
    \textcolor{green}{Green spots}: $HR$ < 1.0.
    (X-axis: Kaplan--Meier survival estimates, with \acrshort{fdr}-adjusted \textit{p}~values (log10 transformed);
y-axis: HR of Cox proportional hazard regression model.)
    }
    \label{fig:hazards3}
\end{figure}

%%
\begin{figure}[H]
% TCGA FDR Pvalue of CAMK2N1 (1.628308e-05), IL19 ( 6.543871e-06), FCGBP (4.827833e-05), CALML5 (0.0001970348)
\setlength{\unitlength}{.78cm}
\begin{picture}(15, 20) %(1,0.55038404)%
\centering
  \put(0,0){\includegraphics[width=14cm]{Figure_4_CAMK2N1_CALML5_FCGBP.pdf}}%
  \put(2.5, 15.3){\fontfamily{qcr}\selectfont
  \tiny *\textit{p} = \num{1.63e-05}}%CAMK2N1
    \put(2.5, 9.25){\fontfamily{qcr}\selectfont
  \tiny *\textit{p} = \num{1.97e-4}}%IL19 6.54e-06 -> CALML5 0.0001970348
    \put(2.5, 2.45){\fontfamily{qcr}\selectfont
  \tiny *\textit{p} = \num{4.83e-05}}%FCGBP

%\begin{annotationimage}{width=15cm}{Figure_4_CAMK2N1_IL19_FCGBP.pdf}
%\includegraphics[width=15cm]{Figure_4_CAMK2N1_IL19.pdf}
%\draw[annotation left = {Aries at 0.3}]

%\end{annotationimage}
\end{picture}%

\bcaption{\hl{Kaplan--Meier survival analyses, by cutoff finding.}}
{The Kaplan--Meier curves of (\textbf{a}) CAMK2N1, (\textbf{c}) CALML5, and (\textbf{e}) FCGBP with optimal \textit{p}~values. 
%((b) the cutoffs derived from it's cumulative \textit{p}~value plot).
%(c) Kaplan--Meier plot of IL19 under optimal \textit{p}~value;
%((d) the cutoffs derived from it's cumulative \textit{p}~value plot).
%(e) Kaplan--Meier plot of FCGBP under optimal \textit{p}~value.\\
%The optimal cutoff value for CAMK2N1 is 
The cutoffs in the cumulative \textit{p}~value plots of (\textbf{b}) CAMK2N1, (\textbf{d}) CALML5, and (\textbf{f}) FCGBP,
show that over 50\% of those unadjusted \textit{p}~values derived by the sliding-window cutoff-finding procedure are below 0.001.
(* \textit{p}: \textit{p}~value adjusted by \acrlong{fdr}, \acrshort{fdr}.)
}
\label{fig:figure4}
\end{figure}



%%
\subsection{GSE65858}

\begin{figure}[H]
   % \centering
    \includegraphics[width=13cm]{Rplot_GSE65858_CoxHR_CAMK2N1_top3FDRKM.pdf}
    \bcaption{\hl{Volcano plot of genes in survival analyses of GSE65858 cohort.}}{
    % GSE117973 using the same platform GPL10558
    This HNSCC cohort was used for filtering of our candidate genes: \textcolor{red}{CAMK2N1}, \textcolor{red}{CALML5}, and \textcolor{red}{FCGBP}.
    In total, 534 genes had \acrshort{fdr}-adjusted \textit{p}~values less than 0.05
    \textcolor{red}{Red spots}: hazard ratios are greater than 1.0;
    \textcolor{green}{Green spots}: hazard ratios are under 1.0.
    The 22 genes, listed on the side, had hazard ratios >$1.8$ or <$0.6$.
    (X-axis: Kaplan--Meier survival estimates, with \acrshort{fdr}-adjusted \textit{p}~values, log10 transformed; y-axis: the hazard ratio (HR) under the Cox proportional hazard regression model).
    }
    \label{fig:hazards534}
\end{figure}

%%



%%
\section{Discussion}
\subsection{The Three Biomarkers in Cancer}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection*{The Protein/Pathology Atlas} %still TCGA's RNA-Seq (ok)

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection*{Literature Review} % literature (ok)

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Feature Selection for Survival Modeling} % Even there are many $X_1 ... X_n$ physical and social features of patients available for survival modelling in the TCGA.

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{The Purpose of Sliding-window Cutoff Selection}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Technical Considerations}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Limitations of the Study}


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}
\begin{figure}[H]
\widefigure
    %\centering
    
    \begin{subfigure}[a]{0.5\textwidth}
    \includegraphics[width=9cm]{RplotH2H_TCGA_GSE65858_CoxHR.pdf}
    \caption{Cox's hazard ratios from TCGA HNSCC and GSE65858 (Pearson's correlation coefficient~\cite{Schober2018}, r = 0.27).}
    \end{subfigure}
%\hfill
    \begin{subfigure}[b]{0.2\textwidth}
    \includegraphics[width=3cm]{Rplot20_correlation_TCGA_GSE65858_CoxHR.pdf}
    \caption{Correlations of Cox's hazard ratios of those 20 significant genes (Pearson's r = 0.68).}
    \end{subfigure}    
    
    \bcaption{\hl{A head-to-head comparison of Cox's hazard ratios from TCGA HNSCC and GSE65858 datasets.}}{TCGA HNSCC and GSE65858 cohorts were applied for identification and validation of the candidate biomarkers in HNSCC.
    (\textbf{a}) A total of 5404 genes had Cox's hazard ratios from TCGA HNSCC and GSE65858 (Pearson's correlation, r = 0.27).
    %\acrshort{fdr}-\textit{p}~values of less than 0.05 in TCGA HNSCC.
    \textcolor{red}{CAMK2N1}, \textcolor{red}{CALML5}, \textcolor{red}{FCGBP}, and 17 other genes (marked in \textcolor{black}{black}) had hazard ratios (HRs) >$1.8$ or <$0.6$.
    \textcolor{red}{Red spots}: $HR$s > 1.0 in TCGA HNSCC.
    \textcolor{green}{Green spots}: $HR$s < 1.0 in TCGA HNSCC.
    Sizes of spots: bigger for Kaplan--Meier \textit{p}~values in TCGA HNSCC. %Please ensure intended meaning is retained.
    (\textbf{b}) The 20 genes were extracted and shown. The hazard ratios of those genes have a moderate correlation between the two cohorts (Pearson's r = 0.68).
    (X-axis: Hazard ratios of Cox proportional hazard regression model from TCGA HNSCC;
    y-axis: Those values from GSE65858; TCGA: \acrlong{tcga}; HNSCC: \acrlong{hnscc}.)
    }
    \label{fig:hazards_head2head_TCGA_GSE65858}
\end{figure}

\clearpage

\subsection{Future Directions in Translational Medicine}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection{Proteomics Validation}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection{Laboratory Validation}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection{Cancer Type-agnostic Study}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection{Holistic Cancer Care} 


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage


\begin{figure}[hbt!]
\centering
\floatbox[{\capbeside\thisfloatsetup{capbesideposition={right,center},capbesidewidth=.35\linewidth,capbesidesep=quad}}]{figure}[\FBwidth]
{\includegraphics[width =\textwidth, height = 0.6\textheight, keepaspectratio]{Figure_5_holisticCare.pdf}}
{\bcaption{The concept of holistic care for \acrshort{hnscc} patients.}
{\\Beyond carcinogenesis: In the mind--brain--body axis, a stressful environment (giant \textcolor{black}{black} arrow) will trigger an emotional response.}}
%The subconscious mind (brain) will release stress hormones and inflammation signals in response to the emotion. The physical body's internal environment (cells) alters epigenetic control in gene regulation and mRNA expression. Over a long incubation time, the tissue/cells will be transformed into dysplasia and then malignancy (e.g., HNSCC) with help from known carcinogens.\\
%Cancer care: Holistic care should take care of cancer patients' spiritual, emotional, physical, and socioeconomic needs. Physical care will be carried out by medication therapy or surgery. After establishing a therapeutic relationship (TR), the physicians' spiritual properties (empathy, sympathy, and compassion) will engage their patients and recover their self-compassion to gain resilience against the disease through their mind--brain--body axis.
%Thus, we suggest that electric healthcare records (EHR) should include physical, pathological, and psychological data, and even more spiritual information. The \acrshort{tcga} might collect those "holistic features"  (\textcolor{green}{green} dashed line) for further study of personalized medicine.}
\end{figure}

\clearpage

\section{Materials and Methods} % 4
\subsection{Patient Cohort} 
\subsubsection{RNA Sequencing Data} 

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsubsection{Clinical Data} 

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Cutoff Finder Core Engine}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Statistical Consideration for Survival Analysis}

\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

\subsection{Biomarker Selection and Validation}


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage


\section*{Conclusions} % 5


\begin{outline}

\1 Aliquam blandit faucibus nisi, sit amet dapibus enim tempus eu

\2 Nulla commodo, erat quis gravida posuere
\1 elit lacus lobortis est, quis porttitor odio mauris at libero
\1 Nam cursus est eget velit posuere pellentesque
\1 Vestibulum faucibus velit a augue condimentum quis convallis nulla gravida

\end{outline}


\clearpage

%------------------------------------------------

\section*{Citations}

An example of the \texttt{\textbackslash cite} command to cite within the presentation:

This statement requires citation \cite{Smith:2012qr}.

\clearpage

%------------------------------------------------
\subsection*{Užití}

\textit{Informační systém nákup letenek} bude použit v interakci se třemi skupinami uživatelů - Administrátor, Zaměstnanec a zákazník. 

\begin{multicols}{3} % Divide text into multiple columns
\mygreen{Sed diam enim, sagittis nec} condimentum sit amet, ullamcorper sit amet libero. \mybrown{Aliquam vel dui orci}, a porta odio. \myred{Nullam id suscipit} ipsum. \myblue{Aenean lobortis} commodo sem, ut commodo leo gravida vitae. Pellentesque vehicula ante iaculis arcu pretium rutrum eget sit amet purus. Integer ornare nulla quis neque ultrices lobortis. Vestibulum ultrices tincidunt libero, quis commodo erat ullamcorper id.
\end{multicols}

\clearpage
%%%%%%%
\thispagestyle{empty} % No slide header and footer

\bibliographystyle{unsrt}
\bibliography{sample}

\clearpage

%------------------------------------------------

\thispagestyle{empty} % No slide header and footer

\begin{tikzpicture}[remember picture,overlay] % Background box
  \node [xshift=\paperwidth/2,yshift=\paperheight/2] at (current page.south west)
    [rectangle,fill,inner sep=0pt,minimum width=\paperwidth,minimum height=\paperheight/2.1,top color=myblue,bottom color=myblue]{}; % Change the height of the box, its colors and position on the page here
\end{tikzpicture}
% Text within the box
\begin{flushright}
  \vspace{1.6cm}
  \color{white}\sffamily
  {\bfseries\Large Comments and Suggestions\par}% Title
  \vspace{0.5cm}
  \normalsize
%  \myauthor\par % Author name
%  \mydate\par % Date
  \myuni\par
  \vfill
\end{flushright}

%----------------------------------------------------------------------------------------
%
\end{document}